### ISSUE TITLE
gnsync on notes with images

### ISSUE THREAD

**Issue by shadiakiki1986** [jeffkowalski/geeknote#80]
Using `gnsync` on notes with images yields empty notes.
I couldn't find anything about the subject in the documentation.
Is this an unsupported feature?

**Comment by mlopezantequera** [comment]
This pull request on the old repository seems to solve the problem (or at least it did back then):

https://github.com/VitaliyRodnenko/geeknote/pull/340

**Comment by jeffkowalski** [comment]
Yes, I pulled that commit into this repository on Feb 10, 2016 (4a8c30b), but I did not add any documentation.
